scylladb/sphinx-scylladb-theme
# Fix alignment expand icons sidebar

The icons to expand the left sidebar were not perfectly aligned.

**Before**

![image](https://user-images.githubusercontent.com/9107969/140715444-1d8590a2-3c42-4dc0-91ac-06ab0db18d59.png)

**Now**

![image](https://user-images.githubusercontent.com/9107969/140715932-a02b337f-9ebe-4517-ae6d-078f83de84a8.png)

## How to test this PR

1. Clone this PR. For more information, see [Cloning pull requests locally](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally).

2. Enter the docs folder, and run:

```
make preview
````

3. Open http://127.0.0.1:5500/ with your favorite browser, and enter an internal page. The expand icons should be aligned with the right sidebar.
